lesson26/src/js/login.js

```diff
@@ -0,0 +1,134 @@
+const userName = document.getElementById("username");
+const password = document.getElementById("password");
+const loginButton = document.getElementById("js-login-button");
+const storage = localStorage;
+
+if (storage.getItem("token")) window.location.href = "./index.html";
+
+
+const constraint = {
+  username: {
+    validation: () => {
+      return isLimitTextLength(userName.value, 16);
```

Comment:
16 is Magic Number

Reply:
As you said, it was difficult to understand what the numbers represented, and I thought it was an unfriendly situation.

Changed to use variables instead of magic numbers.
[e2589ec](https://github.com/sae-github/JS_lesson/pull/30/commits/e2589ec90738d9332230dd9f9eef89b74900bbb1)